src/ucs/sys/sock.c

```diff
@@ -144,29 +161,25 @@ ucs_status_t ucs_socket_connect(int fd, const struct sockaddr *dest_addr)
         }
     } while ((ret < 0) && (errno == EINTR));
 
-    src_addr_size = sizeof(src_addr);
-    ret           = getsockname(fd, (struct sockaddr*)&src_addr, &src_addr_size);
-    if (ret < 0) {
-        ucs_debug("getsockname(fd=%d) failed: %m", fd);
-    }
-
     ucs_debug("connect(fd=%d, src_addr=%s dest_addr=%s): %s", fd,
-              ((ret < 0) ? "-" :
-               ucs_sockaddr_str((struct sockaddr*)&src_addr, src_str,
-                                UCS_SOCKADDR_STRING_LEN)),
+              ucs_socket_getname_str(fd, src_str, UCS_SOCKADDR_STRING_LEN),
               ucs_sockaddr_str(dest_addr, dest_str, UCS_SOCKADDR_STRING_LEN),
               strerror(conn_errno));
+
     return status;
 }
 
 int ucs_socket_is_connected(int fd)
 {
-    struct sockaddr_storage addr;
-    socklen_t sock_addr_len;
+    struct sockaddr_storage peer_addr = {0};
```

Comment:
no need to initialize

Reply:
it was needed to prevent clang false-positive